egs/swbd/s5c/local/chain/run_tdnn_6z.sh

```diff
@@ -55,7 +55,7 @@ final_layer_normalize_target=0.5
 num_jobs_initial=3
 num_jobs_final=16
 minibatch_size=128
-relu_dim=576
+relu_dim=1024
```

Comment:
The goal of this experiment is to use the same model (to the extent possible) as the xent system and see the improvement due to chain objfn. So changing splice indexes in _6z.sh script to 
--splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -7,2 0" and making relu-dim 768 should be sufficient. Ideally we would not even changed relu-dim from 1024 to 768 as we wanted to compare improvement from just changing the objfn. But we do know from previous experiments that models trained with chain objfn overfit, so they should be smaller. So we reduce the number of parameters, to match those of 6z system.

In case you are not convinced you can start another experiment with 
 --splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -7,2 0" and relu-dim which will make the model the same as xent system (except for the number of leaves i.e., output-dimension of the network). We have free GPUs right now, so we can afford the second experiment.